sigal/__init__.py

```diff
@@ -223,3 +223,37 @@ def serve(destination, port, config):
         httpd.serve_forever()
     except KeyboardInterrupt:
         print('\nAll done!')
+
+@main.command()
+@argument('target')
+@argument('keys', nargs=-1)
+@option('-o', '--overwrite', default=False, is_flag=True,
+        help='Overwrite existing .md file')
+def set_meta(target, keys, overwrite=False):
+    """Write image title to corresponding .md file.
```

Comment:
Docstring needs to be updated, mention that target can be a directory or a media file.